pkg/client/edv/client.go

```diff
@@ -90,15 +91,32 @@ func (c *Client) sendPostJSON(objectToMarshal interface{}, endpoint, statusConfl
 
 	defer closeBody(resp)
 
-	if resp.StatusCode == http.StatusBadRequest {
+	switch resp.StatusCode {
+	case http.StatusBadRequest:
 		return "", getError(resp)
-	} else if resp.StatusCode == http.StatusConflict {
+	case http.StatusConflict:
 		return "", errors.New(statusConflictErrText)
+	default:
+		if resp.StatusCode != http.StatusCreated {
+			respText, err := ioutil.ReadAll(resp.Body)
+			if err != nil {
+				return "", fmt.Errorf("unable to read response: %w", err)
+			}
+
+			return "", errors.New(string(respText))
```

Comment:
fmt.Errorf("%s", respText)

Reply:
done. I also updated everywhere else in this project to use fmt.Errorf